data/core/units/elves/Champion.cfg

```diff
@@ -16,7 +16,7 @@
     {AMLA_DEFAULT}
     cost=48
     usage=fighter
-    description= _ "Those few elves who deliberately hone themselves into weapons of war become something which belies the peace-loving reputation of their race. Dedication and skill are matched with purity of form, and the result is swordsmanship to which few can compare. Even their archery, treated as a secondary pursuit, is masterfully executed."
+    description= _ "Elves are typically pacifistic by nature and most will only attack in retaliation for some wrongdoing wrought upon them. However, there are some elves who revel in the glory of battle and deliberately seek combat—purposeful or otherwise—for their own enjoyment. Naturally, most of these warriors live short lives dedicated mostly to honing their swordsmanship in an effort to survive and conquer the battlegrounds. Mere dedication is not enough for most to endure for very long; only the most talented are capable of persisting through constant combat and earning the title of 'Champion'. Having spent their whole lives refining their prowess with the blade, these elves are exceptionally dangerous and should never be underestimated."
```

Comment:
Are they pacifists or are they sorta like pacifists, but not quite? Do they only attack in retaliation or attack only in retaliation? Put another way, might they choose, instead, to withdraw, or negotiation, when they feel they have been wronged?

"Typically pacifists by nature, most elves will attack only in retaliation ..."